pkg/controller/compliancescan/scan.go

```diff
@@ -124,6 +124,41 @@ func newScanPodForNode(scanInstance *compv1alpha1.ComplianceScan, node *corev1.N
 						},
 					},
 				},
+				{
+					Name:  "runtime-kubeletconfig-helper",
+					Image: utils.GetComponentImage(utils.OPERATOR),
+					Command: []string{
+						"sh",
+						"-c",
+						fmt.Sprintf("mkdir -p /host/tmp/compliance-operator && ln -s %s %s | /bin/true", KubeletConfigMapPath, KubeletConfigLinkPath),
```

Comment:
Here we're actually making a change on the node itself. Can't we get this information through the API without having to create symlinks on the actual nodes?

I'm more or less concerned about something happening to this that's outside our control and the feature stops working. Where as if we query an API for the configuration, we have some better guarantees about what's provided.

Reply:
Following up, we need to do this because the scan is technically a node scan, which means the oscap scanner will run with `$HOSTROOT`, which is host specific.

In order for the scanner to scan the file in that location, we need a symlink to make sure it discovers it.

Technically, this will be done once on each scan when this container starts, but it won't clean up the symlink on the host when it's done.